proto/protobuf/repositories.bzl

```diff
@@ -18,34 +18,35 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 load("//proto/protobuf/3rdparty/crates:defs.bzl", "crate_repositories")
 
-def rust_proto_protobuf_dependencies():
-    maybe(
-        http_archive,
-        name = "rules_proto",
-        sha256 = "dc3fb206a2cb3441b485eb1e423165b231235a1ea9b031b4433cf7bc1fa460dd",
-        strip_prefix = "rules_proto-5.3.0-21.7",
-        urls = [
-            "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/refs/tags/5.3.0-21.7.tar.gz",
-            "https://github.com/bazelbuild/rules_proto/archive/refs/tags/5.3.0-21.7.tar.gz",
-        ],
-    )
+def rust_proto_protobuf_dependencies(bzlmod = False):
```

Comment:
Nothing seems to call this with `bzlmod = True` - why do we need to add it?

Reply:
Good catch, thanks. It should have been called with bzlmod=True in the internal deps.